### ISSUE TITLE
Possibility to set speaker distances and levels

### ISSUE THREAD

**Issue by qu15d** [ol-iver/denonavr#269]
I played around with your library and it works great for automation. I'm wondering if it's possible to get/set speaker distances and levels?

**Comment by ol-iver** [comment]
I don't think that it is possible to find speaker distances. At least, I did not find any corresponding command in the [protocol definition](https://www.heimkinoraum.de/upload/files/product/IP_Protocol_AVR-Xx100.pdf).
There are some commands which might set the speaker levels e.g. `PSCEN ?`, but I'm not sure about this.
You can run any command you find in the document via [async_get_command](https://github.com/ol-iver/denonavr/blob/c70e8ce11cd1f9ef86053201588d41d1f63b6e14/denonavr/denonavr.py#L196) or [async_send_telnet_commands](https://github.com/ol-iver/denonavr/blob/c70e8ce11cd1f9ef86053201588d41d1f63b6e14/denonavr/denonavr.py#L200).
`async_get_command` does not return any value, but `async_send_telnet_commands` does.
